fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -187,13 +185,16 @@ def build_download_command(username, password, apple_id, destination = "/tmp", p
 
     def handle_error(password)
       # rubocop:disable Style/CaseEquality
-      unless password === /^[0-9a-zA-Z\.\$\_]*$/
+      # rubocop:disable Style/YodaCondition
+      unless /^[0-9a-zA-Z\.\$\_\-]*$/ === password
```

Comment:
Is there a reason for flopping this comparison around? 🤔

Reply:
Ah okay. I didn't think there was a difference in what was on left/right side but wasn't sure. So this _could_ be changed back and things would still work the same?

Reply:
No. If you change it back, it doesn't work. 
`rubocop` actually wants to write it the other way around, which is probably why this was the other way around and didn't work in the first place - I had to add the `rubocop:disable Style/YodaCondition` to quiet it.